libmambapy/src/libmambapy/bindings/legacy.cpp

```diff
@@ -696,10 +696,42 @@ bind_submodule_impl(pybind11::module_ m)
         .value("CRITICAL", mamba::log_level::critical)
         .value("OFF", mamba::log_level::off);
 
+    py::class_<ChannelContext>(m, "ChannelContext")
+        .def_static(
+            // Still need a singleton as long as mambatest::singleton::channel_context is used
```

Comment:
Wondering whether to leave this, or to take the opportunity to get rid of the ChannelContext singleton altogether.

There aren't that many breakage. The most annoying one is `MatchSpec` which now requires an explicit `ChannelContext` while this is something I wanted to get rid of (but we have so many things to do).

Reply:
I think we can keep it for now, but it would be nice to remove it before releasing 2.0